<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>@danicc097</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>833</width>
     <height>559</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>200</horstretch>
        <verstretch>200</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="rolling">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>795</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>804</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <attribute name="title">
        <string>Rolling stock</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Locomotive simulation</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Locomotive</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>test</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_8">
                    <property name="wrapping">
                     <bool>false</bool>
                    </property>
                    <property name="specialValueText">
                     <string/>
                    </property>
                    <property name="suffix">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Number</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox">
                    <property name="layoutDirection">
                     <enum>Qt::LeftToRight</enum>
                    </property>
                    <property name="wrapping">
                     <bool>false</bool>
                    </property>
                    <property name="specialValueText">
                     <string/>
                    </property>
                    <property name="suffix">
                     <string> u</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Individual mass</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                    <property name="wrapping">
                     <bool>false</bool>
                    </property>
                    <property name="specialValueText">
                     <string/>
                    </property>
                    <property name="prefix">
                     <string/>
                    </property>
                    <property name="suffix">
                     <string> t</string>
                    </property>
                    <property name="maximum">
                     <double>999.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Running mass </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
                    <property name="wrapping">
                     <bool>false</bool>
                    </property>
                    <property name="specialValueText">
                     <string/>
                    </property>
                    <property name="prefix">
                     <string/>
                    </property>
                    <property name="suffix">
                     <string> %</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Front area</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
                    <property name="wrapping">
                     <bool>false</bool>
                    </property>
                    <property name="specialValueText">
                     <string/>
                    </property>
                    <property name="prefix">
                     <string/>
                    </property>
                    <property name="suffix">
                     <string> m²</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Maximum acceleration</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
                    <property name="wrapping">
                     <bool>false</bool>
                    </property>
                    <property name="specialValueText">
                     <string/>
                    </property>
                    <property name="suffix">
                     <string> m/s²</string>
                    </property>
                    <property name="maximum">
                     <double>9.990000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.250000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Wagons</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Front area</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_9">
                    <property name="wrapping">
                     <bool>false</bool>
                    </property>
                    <property name="specialValueText">
                     <string/>
                    </property>
                    <property name="suffix">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Passenger train simulation</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Validate data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>502</height>
        </size>
       </property>
       <attribute name="title">
        <string>Rolling resistance</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="route">
       <attribute name="title">
        <string>Route</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Sim">
       <attribute name="title">
        <string>Simulation</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Window"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGitHub_Homepage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="icon">
    <iconset>
     <normaloff>D:/OneDrive/RailwaySim/Qt/images/png/doc_new_icon&amp;48.png</normaloff>D:/OneDrive/RailwaySim/Qt/images/png/doc_new_icon&amp;48.png</iconset>
   </property>
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>D:/OneDrive/RailwaySim/Qt/images/png/folder_open_icon&amp;48.png</normaloff>D:/OneDrive/RailwaySim/Qt/images/png/folder_open_icon&amp;48.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>D:/OneDrive/RailwaySim/Qt/images/png/save_icon&amp;48.png</normaloff>D:/OneDrive/RailwaySim/Qt/images/png/save_icon&amp;48.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionGitHub_Homepage">
   <property name="icon">
    <iconset>
     <normaloff>D:/OneDrive/RailwaySim/Qt/images/github icon.png</normaloff>D:/OneDrive/RailwaySim/Qt/images/github icon.png</iconset>
   </property>
   <property name="text">
    <string>GitHub Homepage</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset>
     <normaloff>D:/OneDrive/RailwaySim/Qt/images/png/doc_edit_icon&amp;48.png</normaloff>D:/OneDrive/RailwaySim/Qt/images/png/doc_edit_icon&amp;48.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
